k9mail/src/main/java/com/fsck/k9/provider/DecryptedFileProvider.java

```diff
@@ -25,7 +25,7 @@
     private static final long FILE_DELETE_THRESHOLD_MILLISECONDS = 3 * 60 * 1000;
 
 
-    private static boolean receiverRegistered = false;
+    private static DecryptedFileProviderCleanupReceiver receiverRegistered = null;
```

Comment:
The name doesn't fit anymore. Should probably be `cleanupReceiver` or something similar.